packages/relic/lib/src/headers/typed/headers/content_encoding_header.dart

```diff
@@ -79,7 +79,7 @@ class ContentEncoding {
     if (trimmed.isEmpty) {
       throw FormatException('Name cannot be empty');
     }
-    switch (trimmed.toLowerCase()) {
+    switch (trimmed) {
```

Comment:
Question: Is the removal of `toLowerCase` intentional?

Reply:
There are constants like `includeSubDomains` that are case-sensitive. If we lowercase the input string, we would also need to lowercase the constants for the `switch` statement to work properly.

For example:

```dart
static const _includeSubDomains = 'includeSubDomains';
// code...
switch (trimmed.toLowerCase()) {
  case _includeSubDomains:
    return includeSubDomains;
}
// code...
```

The above won't work because `_includeSubDomains` is not lowercased, so it won't match the lowercased `trimmed`.

The only way to handle this would be to explicitly lowercase the constant in the `case`, like so:

```dart
// code...
switch (trimmed.toLowerCase()) {
  case _includeSubDomains.toLowerCase():
    return includeSubDomains;
}
// code...
```

However, this is not possible because `switch` statements require constants, and method calls like `.toLowerCase()` are not allowed.